ibind/base/rest_client.py

```diff
@@ -107,6 +109,15 @@ def __init__(
     def make_logger(self):
         self._logger = new_daily_rotating_file_handler('RestClient', os.path.join(var.LOGS_DIR, f'rest_client'))
 
+
+    def get_headers(        
+        self,    
+        request_method: str,
+        request_url: str,
+        request_params: dict[str, str] | None = None,
+        ):    
+        return None    
+
```

Comment:
Great job implementing this method based on my description 👍

Reply:
The method is nice and clean, but involved and hard to debug. It's not working yet.....

Reply:
> but involved and hard to debug. It's not working yet.....

Yeah, to be honest there's a ton of things they ship for OAuth. You're doing a good job debugging it and pushing it ahead 👍

> If you know of a good way to check headers in a request statement or the response

Can you be more specific? What request would you need to check, at which point? You can amend this line in `RestClient.request` to include headers in the log:
```python
        if log:
            self.logger.info(f'{method} {url} {kwargs}{" (attempt: " + str(attempt) + ")" if attempt > 0 else ""}')
```

And see them in the logged file if you call `ibind_logs_initialize(log_to_file=True)`